src/js/objects/Box.js

```diff
@@ -15,7 +15,7 @@ class Box extends Movable {
 
   isMaterial() {
     var type = this.type.toLowerCase();
-    return (type == "mucosum" || type == "prismatium" || type == "scrapium" || type == "boltrum" || type == "aurus_box");
+    return (type == "mucosum" || type == "prismatium" || type == "scrapium" || type == "boltrum" || type == "aurus_box" || type == "HYBRID_ALLOY_BOX");
```

Comment:
type = this.type.ToLowerCase().,,
So basically convert HYBRID_ALLOY_BOX to hybrid_alloy_box